answer/166ryoryo/JS/05/randomBgColor.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="utf-8">
+<title></title>
+<meta name="keywords" content="">
+<meta name="description" content="">
+<script LANGUAGE="JavaScript">
+  function rangeRandom(num1,num2)	{
+    if((num1 - num2) > 0)	{
+      let big = num1
+      let small = num2
+    }
+    else	{
+      var big = num2
+      var small = num1
+    }
+    let range = big - small + 1
+    let number = Math.floor(Math.random()*range) + small
+    return(number)
+  }
+  function toHex(dec)	{
+    hexChar="0123456789ABCDEF"
+    let i = dec % 16
+    let j = (dec - i)/16
+    result = hexChar.charAt(j)
+    result += hexChar.charAt(i)
+    return result
+  }
+  function ramdomChgBack()	{
+    let backcolor = "#";
+    for(let i = 0; i < 6; i++) {
+      backcolor += (16*Math.random() | 0).toString(16);
```

Comment:
```suggestion
      backcolor += (16 * Math.random() | 0).toString(16);
```
計算の演算子には前後にスペースが必要です